packages/discord.js/typings/index.d.ts

```diff
@@ -2286,8 +2286,11 @@ export class Role extends Base {
   public toString(): RoleMention;
 }
 
-export type SelectMenuType =
-  |;
+export type SelectMenuTypes = ComponentType.StringSelect |
```

Comment:
The name should be `SelectMenuType`. And add it to the `Constents` object.
https://github.com/discordjs/discord.js/blob/89a0ff9783a186461d23c9e25429b49b834ac332/packages/discord.js/typings/index.d.ts#L3195-L3200

Reply:
done